addons/crm_iap_mine/static/src/components/install_module_dialog/install_module_dialog.js

```diff
@@ -0,0 +1,24 @@
+import { Component } from "@odoo/owl";
+import { Dialog } from "@web/core/dialog/dialog";
+
+export class InstallModuleDialog extends Component {
```

Comment:
Reading the purpose of this dialog, I think that we could use directly the `ConfirmationDialog`, making a custom dialog for this seems a bit overkill imo